api/orchestration/v1alpha1/rayclusterfleet_types.go

```diff
@@ -113,6 +113,10 @@ type RayClusterFleetStatus struct {
 	// newest ReplicaSet.
 	// +optional
 	CollisionCount *int32 `json:"collisionCount,omitempty" protobuf:"varint,8,opt,name=collisionCount"`
+
+	// HPAPodSelector for pods that belong to the RayClusterFleet object, this is
+	// needed for HPA to know what pods belong to the RayClusterFleet object.
+	HPAPodSelector string `json:"hpaPodSelector,omitempty"`
```

Comment:
I'd like to double confirm the purpose of this change. Do you mean you like to use naive HPA for `RayClusterFleet` and this will be passed through the HPA object? 

What would be the target? head or entire pods? they both have corresponding labels.

Reply:
could you give an example on the usage?

Reply:
@zhixian82 Thanks for your adoption. yeah, head should be good enough. for multi-node scenarios, only head has the http server running and expose the  application metrics so I can represent the single replica status. If you plan to use resource metrics like SM_ACTIVE or GPU Util, select head and use avg also works. I personally think is not not needed at this moment.

BTW, we have a CN wechat group, if you are interested in it, feel free to send me an email and I can add you in the group (Github profile has the email)

Reply:
Thanks for getting back to me. Aibrix's API design currently supports native HPA, but there's a bug that's preventing HPA from retrieving metric data. Given this, do you think it makes sense to prioritize fixing it?

BTW, I’ve sent you an email. Please add me to the CN WeChat group when you get a chance. Thanks again!

Reply:
talked with @zhixian82 offline, I understand incorrectly earlier and this is a required field for `RayClusterFleet`.